test/msw-api/setup-worker/start/quiet.fn.true.mocks.ts

```diff
@@ -0,0 +1,28 @@
+import { setupWorker, rest } from 'msw'
+
+const worker = setupWorker(
+  rest.get('/user', (req, res, ctx) => {
+    return res(
+      ctx.json({
+        firstName: 'John',
+        age: 32,
+      }),
+    )
+  }),
+  rest.get('/foo', (req, res, ctx) => {
```

Comment:
I'd suggest to use realistic paths so that these usage examples resemble actual usage.